locations/spiders/leroy_merlin_pl.py

```diff
@@ -23,6 +23,10 @@ def parse(self, response):
             # TODO: figure out opening hours
             if item["lat"] < 48.99 or item["lat"] > 54.87 or item["lon"] < 14.07 or item["lon"] > 24.13:
                 # clearly outside Poland, some locations were claimed to be in the Middle East
-                del item["lat"]
-                del item["lon"]
+                if item["lon"] > 48.99 and item["lon"] < 54.87 and item["lat"] > 14.07 and item["lat"] < 24.13:
+                    # swapped lat, lon - swap them back
+                    item["lat"], item["lon"] = item["lon"], item["lat"]
```

Comment:
Is it worth checking the inverse / again after the swap, so that you don't get wrong coordinates and replace them with even worse - ie someone just typo'd something.

Reply:
it IS being checked - this PR adds such line - or at least is supposed to

`if item["lat"] < 48.99 or item["lat"] > 54.87 or item["lon"] < 14.07 or item["lon"] > 24.13:` checks is it outside Poland (rough bbox check)

`if item["lon"] > 48.99 and item["lon"] < 54.87 and item["lat"] > 14.07 and item["lat"] < 24.13:` checks is it more or less inside Poland

Reply:
Oops, missed the above one! Nevermind!